portal-ui/src/screens/Console/Tenants/TenantDetails/AddZoneModal.tsx

```diff
@@ -60,17 +66,15 @@ const AddZoneModal = ({
   classes,
   open,
   onCloseZoneAndReload,
+  volumesPerInstance,
+  volumeSize,
 }: IAddZoneProps) => {
   const [addSending, setAddSending] = useState<boolean>(false);
   const [zoneName, setZoneName] = useState<string>("");
   const [numberOfInstances, setNumberOfInstances] = useState<number>(0);
-  const [volumesPerInstance, setVolumesPerInstance] = useState<number>(0);
-  const [sizeFactor, setSizeFactor] = useState<string>("GiB");
-  const [volumeConfiguration, setVolumeConfig] = useState<string>("");
-  const [storageClass, setStorageClass] = useState<string>("");
 
   const instanceCapacity: number =
-    parseFloat(volumeConfiguration) * volumesPerInstance;
+    volumesPerInstance * volumeSize * volumesPerInstance;
```

Comment:
Is it ok to duplicate `volumesPerInstance`? 
```suggestion
    volumesPerInstance * volumeSize;
```